myia/parser.py

```diff
@@ -262,6 +265,71 @@ def process_Compare(self, block: 'Block', node: ast.Compare) -> ANFNode:
         right = self.process_node(block, node.comparators[0])
         return Apply([ops[0], left, right], block.graph)
 
+    def process_BoolOp(self, block: 'Block', node: ast.BinOp) -> ANFNode:
+        """Process boolean operators: `a and b`, `a or b`."""
+        def fold(block, values, mode='and'):
```

Comment:
This shouldn't have a default value for mode.

Reply:
True.